### DISCUSSION TITLE
Respawning Behavior

### DISCUSSION THREAD

**Discussion by kolakz** [EssentialsX/Essentials#4266] [Ideas and Feature Suggestions] [answered]
Putting the `respawn-listener-priority` to `none` uses vanilla respawning behavior. But if the player hasn't set a respawn point with a bed or home, they would respawn at spawn, but since it uses vanilla respawning behavior, they spawn in a radius around the spawn point rather than on the point directly. So, it would be great for it to use vanilla respawning behavior but if they don't have a home or bed set, it uses EssentialsX respawning behavior, which would make the player respawn exactly at the spawn point.

**Comment by pop4959** [comment, marked as answer]
I'm not really familiar with the dozen spawn options / combinations that Essentials allows, but it sounds like this might be a bug? It probably depends on if the respawn-listener-priority is set to non-none. Maybe you can look into it a bit more and if the behavior is inconsistent please feel free to make an issue.

**Reply by kolakz** [reply]
Will do!
